openmc/mgxs/mgxs.py

```diff
@@ -412,7 +412,7 @@ def get_all_nuclides(self):
         # Otherwise, return all nuclides in the spatial domain
         else:
             nuclides = self.domain.get_all_nuclides()
-            return nuclides.keys()
+            return list(nuclides.keys())
```

Comment:
`keys()`, `values()`, and `items()` are all not `list`s in Python 3, but they are iterable. It seems the change in Python 3.5 is that the interpreter is now a little more strict about what assumptions one can make about dict_keys and dict_values objects.